spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -95,42 +121,42 @@ interface ErrorReceipt {
 
 ### Store Paths
 
-#### Upgrade Sequence Path
+#### Upgrade Channel Path
 
-The upgrade sequence path is a public path that stores the current sequence of the upgrade attempt. The sequence will increment with each attempted upgrade on the given channel. The sequence will be used to ensure that different error receipts referring to different upgrade attempts do not interfere with each other.
+The chain must store the proposed upgrade until the upgrade handshake successfully. This must be stored in the publice store. It may be deleted once the upgrade is successful.
```

Comment:
```suggestion
The chain must store the proposed upgrade upon initiating an upgrade. The proposed upgrade must be stored in the public store. It may be deleted once the upgrade is successful or has been aborted.
```